ibmcloud-powervs/README.md

```diff
@@ -0,0 +1,78 @@
+# Setup instructions
+## Prerequisites
+- golang
+- pvsadm tool
+- IBM Cloud account
+
+> Note: You can only run the below image build instructions on `ppc64le`(CentOS/RHEL). This would additionally require `qemu-img` to be installed.
+
+#### Intall pvsadm
+
+We will be using the tool `pvsadm` to customise and build the VM image. Download or install it using the below instructions.
+
+1. You can download the binary from Github releases [here](https://github.com/ppc64le-cloud/pvsadm/releases)
+   
+2. To install from source
+   ```
+   cd $GOPATH/src/github.com/
+   git clone https://github.com/ppc64le-cloud/pvsadm.git
+   cd pvsadm && make build
+   ```
+
+3. For MacOS, refer [here](https://github.com/ppc64le-cloud/pvsadm#homebrew)
+
+## Image Build
+
+1. Customise the image preparation
+```
+pvsadm image qcow2ova --prep-template-default > image-prep.template
+```
+
+Add the following snippet to `image-prep.template`
+```
+yum install -y gcc gcc-c++ git make
+git clone -b CCv0 https://github.com/kata-containers/kata-containers.git
+git clone -b staging https://github.com/confidential-containers/cloud-api-adaptor.git
```

Comment:
@Amulyam24 - FYI, my colleague @mattarnoatibm spotted that this is using a reference to the staging branch, whereas we have updated to `main` and I this wasn't merged when that update happened.

Reply:
Thanks @stevenhorsman and @mattarnoatibm. I'll submit a fix for it.